### ISSUE TITLE
 SIGSEGV on GCTaskThread markWord::displaced_mark_helper()

### ISSUE THREAD

**Issue by trumple** [adoptium/adoptium-support#852]
### Please provide a brief summary of the bug

[hs_err_pid1.log](https://github.com/adoptium/adoptium-support/files/12241669/hs_err_pid1.log)

### Expected Results

No SIGSEGV 

### Actual Results

SIGSEGV 

### What Java Version are you using?

17.0.8+7

### What is your operating system and platform?

Alpine Linux v3.18 OpenJDK 64-Bit Server VM Temurin-17.0.8+7

### Did it work before?

```Shell
worked on openjdk:11.0.11-slim
```

### Did you test with the latest update version?

```Shell
yes
```

### Did you test with other Java versions?

```Shell
eclipse-temurin:17.0.5_8-jre-alpine also not working
```

**Comment by karianna** [comment]
https://bugs.openjdk.org/browse/JDK-8313658

**Comment by karianna** [comment]
Can you try with -XX:+VerifyBefore and -XX:+VerifyAfterGC flags and get more debug info?

**Comment by trumple** [comment]
[hs_err_pid1.log](https://github.com/adoptium/adoptium-support/files/12250849/hs_err_pid1.log)

In addition when i use -XX:+UseSerialGC the error is no longer present.

**Comment by trumple** [comment]
Moved from Alpine to Ubuntu Jammy resolved the issue

**Comment by karianna** [comment]
OK thanks I'll update the upstream comment but for now I'd recommend sticking with Jammy

**Comment by karianna** [comment]
@trumple we have released 17.0.9 if you wanted to try on Alpine again.
